src/components/Header/Header.component.jsx

```diff
@@ -0,0 +1,39 @@
+import React, {useRef, useState} from 'react';
+import {Col, Container, Row} from 'reactstrap';
+import Burger from '../Burger';
+import DarkMode from '../DarkMode';
+import LoginButton from '../LoginButton';
+import Search from '../Search';
+import Menu from '../Menu';
+import {useOnClickOutside} from '../../utils/hooks/clickOutside';
```

Comment:
for destructuring object elements the styling should be with spaces like:
```suggestion
import { useOnClickOutside } from '../../utils/hooks/clickOutside';
```